### ISSUE TITLE
New Ontology request - GS1

### ISSUE THREAD

**Issue by mobilemadman2** [zazuko/rdf-vocabularies#38]
Hi there.

hey I'm building a very comprehensive seo plugin for wordpress. And I absolutely love this git. So with that said. Ive' been researching new ontologies and specifically GS1 since they've been recognized by schema and W3. But they are not registered on http://dbpedia.org/sparql?nsdecl. But heres what I got for your reference.

https://www.gs1.org/standards/gs1-source/guideline/gs1-source-tsd-technical-implementation-guide-aggregators

https://www.gs1.org/edi-xml/technical-user-guide/Schema_hierarchy

https://www.gs1.se/en/our-standards/Technical-documentation/code-lists/
https://www.gs1.se/esap/20tradeitem/ver285/gu1.htm
https://www.gs1.se/esap/20tradeitem/ver285/gu5.htm
https://github.com/mobilemadman2/gs1

**Comment by ktk** [comment]
@philarcher could you comment on this one, I'm trying to find RDF based vocabs in these links but no luck so far.

**Comment by ktk** [comment]
Ok meanwhile I found https://www.gs1.org/voc/?show=classes

Is there a dump available somewhere containing all classes & properties?

**Comment by philarcher** [comment]
Hi all,

Yes, that's one way to get all the classes, Adrian, but if you view source on any of the individual terms you'll see the whole JSON-LD file (so, for example, view source on https://www.gs1.org/voc/AdditionalProductClassificationDetails).

That's the short term fix.

The better news is that there is a project within GS1 to make a full transition from our existing working method to one that is much more Linked Data/Web friendly. This will see a significant increase in the number of terms in the gs1.org/voc, for example. We're working on the detail and it would be inappropriate to say more here as I may raise false expectations. However, the principles are established and the time line is short - expect to see updates within the coming couple of months.

There's more we're trying to get right. This includes working much more closely with schema.org than we have hitherto, and we have good connections with other relevant departments within search engines/eCommerce sites. So... we're trying. Baby steps and all that. This is a significant cultural change within GS1 and the community it represents so it takes time to make sure everyone's on board.

HTH

Phil

**Comment by ktk** [comment]
@philarcher ok tnx we can work with this for the moment. FYI we will be open sourcing the Schema/Ontology Manager we created for IATA, see https://tcfplayground.org/. It's meant for orgs & non-domain specialists that want to work on a common schema.

**Comment by philarcher** [comment]
Interesting, thanks Adrian. A lot of the ontologies in your tool relate directly to areas where GS1 has a stake in the ground. We have a total of 480 application identifiers, that is, properties, that are designated by numbers including things that map to your https://tcfplayground.org/Dimensions like https://www.gs1.org/standards/barcodes/application-identifiers/3310.

**Comment by mobilemadman2** [comment]
OMG Guys Thanks so much.  Family is going through a tough time.  My wife is dying it's been a nightmare.  I haven't had a chance to go through this in some time.  You guys rock.  Hey, anyone here up for helping me build the Mack daddy of SEO plugins for WordPress.  if interested I've got a lot built and also a basic plugin working but its way too much for the average wp or even the mid-level SEO audience, Only folks like us who understand Structured data can use it.  I want to take what I have and turn it into a very easy and user-friendly build.  I would like to knock out Yoast, it sucks and way too expensive in my mind and SchemaApp again is for folks like us.  Again I have a lot built and I have the overall concept/design/modeling/workflow but I could use some help getting it together.  If interested I would be happy to share in ownership/profits etc.  let me know via email.  drake@brand-x-mobile.com.

**Comment by ktk** [comment]
Sorry to hear that, all the best for you and your family Drake.

I'm afraid we can't help much on the plugin. We don't know a thing about WordPress, our focus is on the RDF technology stack & things like JSON-LD. I will keep an eye on what Phil is doing so we can integrate GS1 as soon as I can fetch it properly from somewhere.

**Comment by ktk** [comment]
@philarcher is there any update on a version that we can source somehow meanwhile?

**Comment by philarcher** [comment]
Hi @ktk 

All the above remains true (and I admit, after all this time, it had quite slipped my mind). The good news is that the work I alluded to has moved up the priority list in recent months - we really *are* working on it in terms of engineering. In general, the visibility of the Web Voc has increased across the organization and we have a significant project to 'semantify' a lot of what we do, including a lot more of the terms used by industry. We had a session devoted to the topic in our big annual event just this week - that hasn't happened before.

We keep a copy of the GS1 Web Voc at https://github.com/gs1/WebVoc these days. We're very aware that we'd like to cover more sectors than just food and apparel - that's been on the to do list since before I joined GS1 nearly 4 years ago :-( 

If there's something I can do in the meantime, shout in my general direction.
